### PR TITLE
Fix undefined constant WC warning and bump WC compatibilty version

### PR DESCRIPTION
**Pull request by jorgemd24**
### Changes proposed in this Pull Request:

This PR fixes the warning when WC is deactivated and the Google Analytics extension is enabled. Also, it bumps the latest WC-compatible version 7.3.

`Warning: Use of undefined constant WOOCOMMERCE_VERSION - assumed 'WOOCOMMERCE_VERSION' (this will throw an Error in a future version of PHP)...`

### Checks:

* [X] Does your code follow the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards?
* [ ] Have you written new tests for your changes, as applicable?
* [X] Have you successfully run tests with your changes locally?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Detailed test instructions:

1. Disable the WC plugin, and the following notice should be displayed: `WooCommerce Google Analytics requires WooCommerce version 6.8 or higher.`
2. No warning should be displayed.

### Changelog entry

> Fix - undefined WC constant.

### CODE REVIEW

File: woocommerce-google-analytics-integration.php [hunk 1]
```diff
@@ -142,9 +142,16 @@ public function load_plugin_textdomain() {
...
-			$error = sprintf( 'WooCommerce Google Analytics requires WooCommerce version %1$s or higher. You are using version %2$s', WC_GOOGLE_ANALYTICS_INTEGRATION_MIN_WC_VER, WOOCOMMERCE_VERSION );
-			echo '<div class="error"><p><strong>' . wp_kses_post( $error ) . '</strong></p></div>';
+			if ( defined( 'WOOCOMMERCE_VERSION' ) ) {
+				/* translators: 1 is the required component, 2 the Woocommerce version */
+				$error = sprintf( 'WooCommerce Google Analytics requires WooCommerce version %1$s or higher. You are using version %2$s', WC_GOOGLE_ANALYTICS_INTEGRATION_MIN_WC_VER, WOOCOMMERCE_VERSION );
+				echo '<div class="error"><p><strong>' . wp_kses_post( $error ) . '</strong></p></div>';
+			} else {
+				/* translators: 1 is the required component */
+				$error = sprintf( 'WooCommerce Google Analytics requires WooCommerce version %1$s or higher.', WC_GOOGLE_ANALYTICS_INTEGRATION_MIN_WC_VER );
+				echo '<div class="error"><p><strong>' . wp_kses_post( $error ) . '</strong></p></div>';
```
Comment (mikkamp) [inline woocommerce-google-analytics-integration.php]: I would leave the echo outside of the if statement, no need to duplicate it twice.

File: woocommerce-google-analytics-integration.php [hunk 2]
```diff
@@ -142,9 +142,16 @@ public function load_plugin_textdomain() {
 		 * WooCommerce fallback notice.
 		 */
 		public function woocommerce_missing_notice() {
-			/* translators: 1 is the required component */
-			$error = sprintf( 'WooCommerce Google Analytics requires WooCommerce version %1$s or higher. You are using version %2$s', WC_GOOGLE_ANALYTICS_INTEGRATION_MIN_WC_VER, WOOCOMMERCE_VERSION );
-			echo '<div class="error"><p><strong>' . wp_kses_post( $error ) . '</strong></p></div>';
+			if ( defined( 'WOOCOMMERCE_VERSION' ) ) {
+				/* translators: 1 is the required component, 2 the Woocommerce version */
```
Comment (mikkamp) [inline woocommerce-google-analytics-integration.php:146]: I'm ok with leaving this comment, but the string can't actually be translated unless we wrap it in a `__()` function call

**Review by mikkamp — APPROVED** [review]
Thanks the changes look good. I just added some minor comments, but I'll go ahead and approve it.

**Comment by jorgemd24** [comment]
Thanks, @mikkamp for looking at this. I addressed your comments 👍

**Pull request merged**